elixir/apps/domain/priv/repo/migrations/20241126185037_add_identity_email_unique_index.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Domain.Repo.Migrations.AddIdentityEmailUniqueIndex do
+  use Ecto.Migration
+
+  def change do
+    create(
+      index(:auth_identities, [:provider_id, :email],
```

Comment:
We might have a convention to include the account_id in these

Use case might be to speed up queries where we need to filter by account_id such as browsing data in metabase

```suggestion
      index(:auth_identities, [:account_id, :provider_id, :email],
```

Reply:
Updated the migration with this suggestion